src/txn/txn_rollback_to_stable.c

```diff
@@ -8,6 +8,9 @@
 
 #include "wt_internal.h"
 
+/* Enable rollback to stable verbose messaging during recovery*/
```

Comment:
please add a space between the last word and the trailing "*/". Also a full stop at the end of the statement.